### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by smacsx**
For code review

### CODE REVIEW

File: Sample.R [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# setwd("SET THE Working Director to THE PATH TO THIS DIRECTORY")
+rm(list = ls())
```
Comment (zaizennu) [inline Sample.R:2]: ampanget

File: Sample.R [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+# https://www.r-bloggers.com/evaluating-logistic-regression-models/
+
+# If you have some trouble with packages on MacOS use the following link 
+# https://swvanderlaan.github.io/post/getting-r-with-tcl-tk-on-my-mac/
+
+library(aod)
```
Comment (zaizennu) [inline Sample.R:9]: what the duck is this?

**Review by zaizennu — COMMENTED** [review]
Do well next time

File: Sample.R (hunk 1 above)
Comment (zaizennu) [reply Sample.R:2]: wqeqwe

Comment (smacsx) [reply Sample.R:2]: f6r6fr6frrfrd

Comment (zaizennu) [reply Sample.R:2]: ### Hello

> This is wrong
`m = eweqwe`

**Review by zaizennu — APPROVED** [review]

**Pull request merged**